src/actions/pause/continueToHere.js

```diff
@@ -36,7 +34,7 @@ export function continueToHere(line: number) {
       addHiddenBreakpoint({
         line,
         column: undefined,
-        sourceId: source.id
+        sourceId: getSelectedSource(getState()).id
```

Comment:
if there isn't a source, this will fail.

Perhaps we get the source above and if there isn't one we return early?